pandas/core/groupby/grouper.py

```diff
@@ -800,7 +800,7 @@ def get_grouper(
 
     # what are we after, exactly?
     any_callable = any(callable(g) or isinstance(g, dict) for g in keys)
-    any_groupers = any(isinstance(g, Grouper) for g in keys)
+    any_groupers = any(isinstance(g, Grouper) or isinstance(g, Grouping) for g in keys)
```

Comment:
how is this actually hit?

Reply:
`isinstance(g, Grouper) or isinstance(g, Grouping)` -> `isinstance(g, (Grouper, Grouping))` pls

Reply:
L1692:1697 of generic in this PR is passing in a list of groupings.

Reply:
It is. I have improved the code and added a test (`test_ambiguous_grouping()`).